### PR TITLE
Consolidate form-field-directives

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: docs/src/main/paradox/scala/http/routing-dsl/directives/form-field-directives/index.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
 @@@ index
 
 * [formField](formField.md)
-* [formFields](formFields.md)
+* @scala[[formFields](formFields.md)]
```
Comment (ktoso) [inline docs/src/main/paradox/scala/http/routing-dsl/directives/form-field-directives/index.md]: does this look fine or shows empty `*` in java?

Comment (jlprat) [reply docs/src/main/paradox/scala/http/routing-dsl/directives/form-field-directives/index.md]: The link `FormFields` doesn't appear in the Java version

Comment (ktoso) [reply docs/src/main/paradox/scala/http/routing-dsl/directives/form-field-directives/index.md]: I know the *link* won't, I'm worried if an empty:

- formField
-
- other thing

will appear for java

Comment (jlprat) [reply docs/src/main/paradox/scala/http/routing-dsl/directives/form-field-directives/index.md]: This is how it appears, no empty space :)
selection_006

**Pull request merged**